compiler/rustc_middle/src/ty/context.rs

```diff
@@ -2,6 +2,8 @@
 
 #![allow(rustc::usage_of_ty_tykind)]
 
+pub mod tls;
```

Comment:
Does this need to be inside the `context` module, or can it be inside `ty`?

Reply:
It accesses private fields from `TyCtxt`, so it makes sense to keep it there.